### PR TITLE
Disable active storage for Rails 6.1.0

### PR DESCRIPTION
**Pull request by gonuguntla**

### CODE REVIEW

File: config/application.rb [hunk 1]
```diff
@@ -1,8 +1,20 @@
 # frozen_string_literal: true
 
-require File.expand_path('../boot', __FILE__)
+
+require_relative 'boot'
 
 require 'rails/all'
+require "active_model/railtie"
```
Comment (thisisstephenbetts) [inline config/application.rb:7]: Looks fine but why does this need to change?

File: db/migrate/20210226185223_create_active_storage_tables.active_storage.rb [hunk 2]
```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+# This migration comes from active_storage (originally 20170806125915)
+class CreateActiveStorageTables < ActiveRecord::Migration[5.2]
```
Comment (thisisstephenbetts) [inline db/migrate/20210226185223_create_active_storage_tables.active_storage.rb:4]: Why do we need ActiveStorage tables if we're using S3? I'd have assumed one or the other (and I'd probably go with the DB for simplicity for now, personally)

**Review by thisisstephenbetts — COMMENTED** [review]
Got a couple of questions. Not sure why we need S3 & DB for ActiveStorage. We can chat about it on a call or on Slack if you like.

File: config/application.rb (hunk 1 above)
Comment (thisisstephenbetts) [reply config/application.rb:7]: Why go from `rails/all` I mean. I know there can be reasons, just wondering

Comment (gonuguntla) [reply config/application.rb:7]: active_storage issue we can solve  in 2 ways

1) Install active storage  and add db migrations, configs

2) Disable active storage.
we do not want to load all  "rails/all".

**Review by thisisstephenbetts — APPROVED** [review]
Looks good — that makes sense. Thanks Deepika!

**Pull request merged**